impl/client/src/main/doc-js/pentaho/context/IContext.jsdoc

```diff
@@ -15,14 +15,14 @@
  */
 
 /**
- * The `IContext` interface exposes contextual information of the Pentaho Web Client Platform,
- * like, for example, its
+ * The `IContext` interface allows access to the Pentaho Web Client Platform.
```

Comment:
Thanks for the feedback :)
this was an attempt of mine, to make this uniform with doc suggestion in the other two files, I will change this after feedback from the doc team to a suggestion I gave taking into account what you said